Starbust.md

```diff
@@ -0,0 +1,183 @@
+# Fraud detection in RHODS using Starbust Enterprise
+
+##### Requirements:
+
+- RHODS working cluster with administrator access
+- Startbust Enterprise licence
+- Write access to your own Amazon S3 bucket
+- Access to the [original dataset](https://drive.google.com/file/d/1YhmV3vPbFe-JXU_biwvaizV0WGhAegH1/view)
+
+#### What is Trino and Starbust Enterprise?
+
+Trino (formerly Presto® SQL) is the fastest open source, massively parallel
+processing SQL query engine designed for analytics of large datasets distributed
+over one or more data sources in object storage, databases and other systems.
+
+Starburst Enterprise platform (SEP) is a fully supported, enterprise-grade
+distribution of Trino. It adds integrations, improves performance, provides
+security, and makes it easy to deploy, configure and manage your clusters.
+
+### Goal
+
+In this document we will cover how to work with the data of the fraud-detection
+example with SQL-like queries and an instance of SEP running in the Red Hat
+Openshift Data Science cluster.
+
+### Steps
+
+#### Store the original data in S3
+
+Upload the
+file [creditcard_with_empty_values.csv]('https://drive.google.com/file/d/1YhmV3vPbFe-JXU_biwvaizV0WGhAegH1/view?usp=share_link')
+to a bucket called `rhods-fraud-detection/data`.
```

Comment:
Usually aws s3 bucket names are unique globally therefore may be use a template here link PASTE-YOUR-BUCKET-NAME or something similar